packages/react-query/package.json

```diff
@@ -45,14 +45,14 @@
       "react-hooks/exhaustive-deps": "error",
       "no-restricted-imports": [
         "error",
-        "@trpc/react"
+        "@trpc/react-query"
       ]
     }
   },
   "peerDependencies": {
     "@tanstack/react-query": "^4.3.8",
-    "@trpc/client": "^10.0.0-alpha.48",
-    "@trpc/server": "^10.0.0-alpha.48",
+    "@trpc/client": "^10.0.0-proxy-beta.21",
+    "@trpc/server": "^10.0.0-proxy-beta.21",
```

Comment:
same as above, we want versions to be synced and since `@trpc/react-query` won't exist before .21 we should probably have these like this?

Reply:
@sachinraja told me to not bump up the version so I'm not sure.

Reply:
Thats all the other ones that are done automatically prepublish. The peerDeps we need to do ourselves